bears/python/RadonBear.py

```diff
@@ -17,39 +16,32 @@ class RadonBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Complexity'}
 
-    def run(self, filename, file,
-            radon_ranks_info: typed_list(str)=(),
-            radon_ranks_normal: typed_list(str)=('C', 'D'),
-            radon_ranks_major: typed_list(str)=('E', 'F')):
+    def run(self, filename, file, cc_threshold: int=10):
         """
         Uses radon to compute complexity of a given file.
 
-        :param radon_ranks_info:   The ranks (given by radon) to
```

Comment:
this breaks backwards compatibility with existing `.coafile` settings people might have.  cc @sils to ensure he thinks that is necessary in this case.

Reply:
why can't we have the old settings there for one release and warn people if they're used, falling back to the old way?

Reply:
that sounds good.